app/controllers/rooms_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class RoomsController < ApplicationController
+  before_action :set_room, only: %i[show update]
+  # GET /rooms/:room_hash
+  def show
+    @jams = @room.jams.order('jams.created_at DESC').to_a
+    @current_jam = @jams.shift
+  end
+
+  # POST /rooms
+  def create
+    @room = Room.new
+
+    @room.save
```

Comment:
if there is really no validation of any kind on room, than it should be just `room = Room.create` (notice no `@` too)